config/kube.py

```diff
@@ -270,7 +392,7 @@ def ip(vip):
     if not netaddr.IPAddress(vip):
         click.echo('Invalid IP address %s' % vip)
         return
-    _update_kube_server('IP', vip)
+    _update_kube_server(KUBE_SERVER_IP, vip)
 
```

Comment:
Add comment about previously merged lines 276-289-  `config kube server insecure on/off` and `config kube server disable on/off` Why use on/off instead of true/false, as the values are displayed in output of `show kube server` ? It was unintuitive to me at first

Reply:
Me too. Per @jleveque comments, I switched to on/off. He would know the reason behind.